resources/views/app/includes/footer.blade.php

```diff
@@ -8,7 +8,7 @@
 
             <div>
                 <span class="fa fa-phone"></span>
-                +44 (0) 1225 383067
+                (01225) 666076
```

Comment:
Well, this sounds like a very sucky situation :slightly_frowning_face: 

Have you checked whether anywhere else needs updating?

Reply:
Good shout: I looked at the "General Enquiries" tab of the "Contact" page, and the number is written there, too, so I'll put in a PR for that as well.

Reply:
I've never used any of the automated features (Event Reporting) and am unsure of how they work in the back-end, so I don't know if the phone number is listed in any of these areas.

Reply:
Can you keep them all in the same PR? Quicker/easier to deploy :slightly_smiling_face: 

> I don't know if the phone number is listed in any of these areas.

I doubt the backend will reference the phone number (it only sends emails) but a quick search of the number (eg, 383067) should show you everywhere it's used.

Reply:
Should both be in this PR now :)

Reply:
Are these the only 2 to change? If so, I'll :heavy_check_mark: and merge :slightly_smiling_face:

Reply:
Yes thank you; I couldn't find any other mentions of the number.